docs/source/tutorials/index.rst

```diff
@@ -15,3 +15,4 @@ Tutorials
    token_classification
    pred_probs_cross_val
    faq
+   regression
```

Comment:
Lets put this under token-classification

Reply:
rearranged in [5aee141](https://github.com/krmayankb/cleanlab/pull/1/commits/5aee14192d6dee8f5340adadb50331e55079b140)